src/main/resources/db/changelog/changes/03-create-roles-table.yaml

```diff
@@ -0,0 +1,22 @@
+databaseChangeLog:
+  - changeSet:
+      id: create-roles-table
+      author: nshtykh
+      changes:
+        - createTable:
+            tableName: roles
+            columns:
+              - column:
+                  name: id
+                  type: bigint
+                  autoIncrement: true
+                  constraints:
+                    primaryKey: true
+                    nullable: false
+              - column:
+                  name: name
+                  type: varchar(255)
+                  constraints:
+                    nullable: false
+                    unique: true
+                    
```

Comment:
I guess you missed an empty line at the end of the file
<img width="344" alt="Screenshot 2023-12-25 at 09 34 14" src="https://github.com/nshtykh/online-book-store/assets/45499568/fc062479-64cc-46c2-95a7-32c873984744">

Reply:
agree